src/libcompiler_builtins/lib.rs

```diff
@@ -180,7 +180,7 @@ pub mod reimpls {
             sr = sr.wrapping_add(1);
 
             // 1 <= sr <= u64::bits() - 1
```

Comment:
And the C code is actually assuming it when executing the shift, it sets the low value to 0 because it knows that n gets shifted at least by 65 bits.